src/envoy/alts/alts_socket.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Istio Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto3";
+
+package envoy.security.v2;
+
+import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+
+message AltsSocket {
+  string handshaker_service = 1 [(validate.rules).string.min_bytes = 1];
```

Comment:
Could you add some comments to explain this field? Is it used specify tls/loas handshake?